### ISSUE TITLE
[facets] Unicode support for ID3 tags

### ISSUE THREAD

**Issue by foxbunny** [Outernet-Project/librarian-content#8]
Hachoir does not seem to support Unicode strings in ID3 tags. Mutagen does nto seem to have this problem, so might be worth a look.

http://mutagen.readthedocs.org/en/latest/api/id3.html

```
>>> m = mutagen.id3.ID3('01 - ファイナルダーリン.mp3')
>>> m.keys()
[u'COMM:iTunes_CDDB_IDs:eng', u'COMM:iTunPGAP:eng', 'TPOS', u'APIC:', 'TPUB', u'COMM:iTunSMPB:eng', 'TIT2', 'TDRC', 'TENC', u'COMM::eng', 'TRCK', u'COMM:iTunNORM:eng', 'TPE1', 'TALB', 'TSRC', 'TCON', 'TPE2']
>>> print(m['TPE2'].text[0])
やくしまるえつこ
>>> print(m['TIT2'].text[0])
ファイナルダーリン
>>> 
```

Another candidate is eyed3: http://eyed3.nicfit.net/

```
>>> import eyed3
>>> x = eyed3.load('01 - ファイナルダーリン.mp3')
>>> x.tag.artist
u'\u3084\u304f\u3057\u307e\u308b\u3048\u3064\u3053, D.V.D'
>>> print(x.tag.artist)
やくしまるえつこ, D.V.D
>>> 
```

**Comment by foxbunny** [comment]
Mutagen is also able to parse mp4 metadata:

```
>>> import mutagen.mp4
>>> x = mutagen.mp4.MP4('00002-Primer on the unemployment rate.mp4')
>>> x.keys()
['\xa9ART', '\xa9day', '\xa9too', '\xa9nam', '\xa9cmt', 'desc']
>>> x['desc']
[u'Understanding how the headline unemployment rate (U-3) is calculated\nMore free lessons at: http://www.khanacademy.org/video?v=C9onjpQGpSg']
>>> x['\xa9ART']
[u'Khan Academy']
>>> 
```

**Comment by goodbadwolf** [comment]
Resolved in 15360b01d04531acc9f5af5a945fc49e7b566197
